python/metatensor-torch/tests/atomistic/io.py

```diff
@@ -0,0 +1,73 @@
+import os
+
+import pytest
+import torch
+
+import metatensor.torch
+from metatensor.torch import Labels, TensorBlock
+from metatensor.torch.atomistic import NeighborListOptions, System, load, save
+
+
+@pytest.mark.parametrize(
+    "pbc", [[True, True, True], [True, False, True], [False, False, False]]
+)
+@pytest.mark.parametrize("nl_size", [10, 100, 1000])
+@pytest.mark.parametrize("cutoff", [3.5, 5.0])
+@pytest.mark.parametrize("full_list", [True, False])
+@pytest.mark.parametrize("strict", [True, False])
+@pytest.mark.parametrize("extra_data", [False, True])
+@pytest.mark.filterwarnings("ignore:custom data")
```

Comment:
I don't think it makes sense to parametrize on everything here, it will just make the tests slower without bringing extra checks. I would rather do one test saving everything possible at the same time.